### PR TITLE
Junaid/1xlibsfix

### LINKED ISSUE #3823: 1 low severity vulnerability
#### Expected behavior
Don't know
#### Actual behavior
$ npm audit fix

up to date, audited 351 packages in 3s

44 packages are looking for funding
  run `npm fund` for details

# npm audit report

web3  *
Insecure Credential Storage - https://npmjs.com/advisories/877
No fix available
node_modules/web3

1 low severity vulnerability

Some issues need review, and may require choosing
a different dependency.

#### Steps to reproduce the behavior

1.$ npm install web3 (which gives this message but asks for audit fix)
2. npm audit fix (which again generates the problem message

#### Logs

no log file generated
#### Environment
Ubuntu 20.04 virtual machine running under ubuntu 18.04
npm@7.0.15 /home/osboxes/.nvm/versions/node/v15.4.0/lib/node_modules/npm
Laptop: amd64bit machine

### LINKED ISSUE #4225: 1.x Fix libs with vulnerabilities
- Fix all libs with **High** severe vulnerabilities

### PR DESCRIPTION
**Pull request by jdevcs**
## Description

Please include a summary of the changes and be sure to follow our [Contribution Guidelines](../CONTRIBUTIONS.md).

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I ran `npm run dtslint` with success and extended the tests and types if necessary.
- [x] I ran `npm run test:unit` with success.
- [x] I ran `npm run test:cov` and my test cases cover all the lines and branches of the added code.
- [x] I ran `npm run build` and tested `dist/web3.min.js` in a browser.
- [ ] I have tested my code on the live network.
- [ ] I have checked the Deploy Preview and it looks correct.
- [x] I have updated the `CHANGELOG.md` file in the root folder.

### CODE REVIEW

**Review by luu-alex — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -425,6 +425,12 @@ Released with 1.0.0-beta.37 code base.
...
 - Support for EIP-1559 to `web3.eth.sendTransaction` (#4220)
 
+## [1.5.2]
+
 ## [Unreleased]
 
-## [1.5.2]
+### Changed
+
+- ethers from 5.1.4 to 5.4.4
```
Comment (Spacesai1or) [inline CHANGELOG.md]:
```suggestion
- ethers from 5.1.4 to 5.4.4 (#4231)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -425,6 +425,12 @@ Released with 1.0.0-beta.37 code base.
...
+- karma from 5.2.3 to 6.3.4
```
Comment (Spacesai1or) [inline CHANGELOG.md]:
```suggestion
- karma from 5.2.3 to 6.3.4 (#4231)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -425,6 +425,12 @@ Released with 1.0.0-beta.37 code base.
...
+- lerna from 3.22.1 to 4.0.0
```
Comment (Spacesai1or) [inline CHANGELOG.md]:
```suggestion
- lerna from 3.22.1 to 4.0.0 (#4231)
```

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -46,7 +46,7 @@ jobs:
         runs-on: ubuntu-latest
         strategy:
             matrix:
-                node: [8, 10, 11, 12, 13]
+                node: [11, 12, 13, 14]
```
Comment (Spacesai1or) [inline .github/workflows/build.yml:49]: Should we mention in the `CHANGELOG.md` (and eventually the release notes) that we're dropping testing for Node versions 8 and 10, and are also now testing for support for v14

**Review by Spacesai1or — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 4 above)
Comment (jdevcs) [reply .github/workflows/build.yml:49]: yes, added in changelog

**Review by Spacesai1or — APPROVED** [review]

**Pull request merged**